ck3-tiger/src/main.rs

```diff
@@ -192,9 +201,17 @@ fn main() -> Result<()> {
         bail!("Cannot find mod directory. Please make sure the .mod file is correct.");
     }
     eprintln!("Using mod directory: {}", modpath.display());
+    eprintln!();
```

Comment:
I think this has to be moved down a bit, because the Everything module does eprintln when loading extra mods. This happens while loading the config file.